src/font/text.cpp

```diff
@@ -337,19 +360,41 @@ bool pango_text::set_text(const std::string& text, const bool markedup)
 					<< " text '" << text
 					<< "' contains invalid utf-8, trimmed the invalid parts.";
 		}
+
+		if (highlight_start_offset_ != highlight_end_offset_) {
+			/** Highlight */
+			PangoAttrList *attribute_list = pango_attr_list_new();
+			int col_r = highlight_color_.r / 255.0 * 65535.0;
+			int col_g = highlight_color_.g / 255.0 * 65535.0;
+			int col_b = highlight_color_.b / 255.0 * 65535.0;
+			DBG_GUI_D << "highlight start : " << highlight_start_offset_ << "end : " << highlight_end_offset_;
+			DBG_GUI_D << "highlight rgb : " << col_r << "," << col_g << "," << col_b;
+			PangoAttribute *attr = pango_attr_background_new(col_r, col_g, col_b);
+			attr->start_index = highlight_start_offset_;
+			attr->end_index = highlight_end_offset_;
+			pango_attr_list_insert(attribute_list, attr);
+
+			pango_layout_set_attributes(layout_.get(), attribute_list);
+			//pango_attr_list_unref(attribute_list);
```

Comment:
If this isn't a "todo: check if needed" then it should be deleted.

Reply:
Removed.